web-admin/src/components/layout/ContentContainer.svelte

```diff
@@ -9,7 +9,7 @@
 >
   <section class="w-full flex flex-col gap-y-3" style:max-width="{maxWidth}px">
     {#if title && showTitle}
-      <h1 class="text-2xl font-semibold">
+      <h1 class="text-2xl font-semibold" aria-label="Container title">
```

Comment:
Typically aria-labels are only applied on interactive elements. To access this element in Playwright, what about
```
await page.getByRole('heading', { level: 1, name: '<container-title>' })
```

Reply:
Hmmm I tried a bit to get this to work. But it always timed out for me. Lets not spend too much time here, we can get back to this some other time.